src/main/java/lotto/util/RandomUtil.java

```diff
@@ -0,0 +1,13 @@
+package lotto.util;
+
+import java.util.Random;
+
+public class RandomUtil {
+
+    private static final int BOUND = 45;
+    private static final Random random = new Random();
+
+    public static int getRandom() {
+        return random.nextInt(BOUND) + 1;
```

Comment:
+1을 하기 보단, range를 주면 어떨까요?

```suggestion
        return random.nextInt(1, BOUND + 1);
```
예를들어 위와 같이요.